### PR TITLE
MP-33 Create Skeleton component

### PR DESCRIPTION
**Pull request by DiegoVillasenorVilla**
### Description

- I created the skeleton component with the corresponding animation.
- I added to to component the next props:
  - width: string
  - height: string
  - type: Typo
- I created the type Typo that have two strings = 'circle' and 'container'

image
image

### Testing Checklist

- [x] Test cases have been added/updated
- [x] Tested locally in browser (at least one other than Chrome)
- [x] All tests pass

### What JIRA Ticket(s) does this PR address?

- [MP-33](https://cd95.atlassian.net/browse/MP-33)

### CODE REVIEW

File: src/components/linearProgressBar/LinearProgressBar.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```
Comment (caarlosdamian) [inline src/components/linearProgressBar/LinearProgressBar.tsx:1]: this are not related to skeleton

File: src/components/skeleton/skeleton.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  }
+}
+
+.circle {
+  background: #E6E9EF;
+  border-radius: 100px;
+}
+
+.container {
+  background: #E6E9EF;
```
Comment (caarlosdamian) [inline src/components/skeleton/skeleton.css:20]: move this to .skeleton no need to duplicate it

File: src/components/skeleton/skeleton.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  0% {
+    background-color: hsl(200, 20%, 70%);
+  }
+  100% {
+    background-color: hsl(200, 20%, 95%);
+  }
+}
+
+.circle {
+  background: #E6E9EF;
```
Comment (caarlosdamian) [inline src/components/skeleton/skeleton.css:15]: move this to .skeleton no need to duplicate it

File: src/components/skeleton/skeleton.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  border-radius: 100px;
```
Comment (caarlosdamian) [inline src/components/skeleton/skeleton.css:16]: change this to 50%

**Review by caarlosdamian — CHANGES_REQUESTED** [review]
some comments

File: src/components/linearProgressBar/LinearProgressBar.tsx (hunk 1 above)
Comment (DiegoVillasenorVilla) [reply src/components/linearProgressBar/LinearProgressBar.tsx:1]: ok, but I don´t remember to commited that

**Review by caarlosdamian — APPROVED** [review]
lgtm
